src/kernel/arch/aarch64/arch.zig

```diff
@@ -113,6 +115,74 @@ pub fn spinWait() noreturn {
     while (true) {}
 }
 
+pub const cpu = struct {
```

Comment:
This is really nice. We could consider using this in x86 in the future! Don't forget to capitalise `Cpu`.

Reply:
Thanks. Done.